packages/web/src/app/web-player/WebPlayer.jsx

```diff
@@ -725,6 +725,17 @@ const WebPlayer = (props) => {
                 path={COIN_DETAIL_PAGE}
                 isMobile={isMobile}
                 render={(props) => {
+                  const ticker = props.match.params.ticker
```

Comment:
can we not just always toUpperCase it? like replace props on line 739 with an obj that includes the toUpperCase'd ticker